index.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-alert */
+/* eslint-disable indent */
+/* eslint-disable no-multi-spaces */
+/* eslint-disable object-curly-spacing */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable object-curly-newline */
+
```

Comment:
- [ ] The purpose of linters in a project is to write maintainable and standard code across a project. Kindly remove these linter disablings here.